backend/contentBackend.js

```diff
@@ -17,15 +17,15 @@ function cleanRoute(someRoute) {
 }
 
 /**
- * Authenticate a user using jwt, and then validate
+ * Authenticate a user and then validate
  * the provided route by checking for null/undefined.
  *
- * @param { string } jwt - token used for identification
  * @param { string } route - site route that will be validated
  * @param { string } invalidRouteError - error that should be thrown on invalid route
  */
-async function authUserAndValidateRoute(jwt, route, invalidRouteError) {
-  await validateJWT(jwt);
+async function authUserAndValidateRoute(route, invalidRouteError) {
```

Comment:
Suggestion: just call this function `validateRoute` and remove the next line.  Then instead of calling `authUserAndValidateRoute`, you can
```ts
await validateUser();
validateRoute(route, 'error message');
```
and you've typed 4 extra characters in exchange for code that's easier to read.  (You could have the same number of characters if you called the other function `authUser` rather than `validateUser`.)